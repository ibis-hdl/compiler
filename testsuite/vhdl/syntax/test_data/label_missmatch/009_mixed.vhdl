library IEEE;
use IEEE.STD_LOGIC_1164.all;
use IEEE.NUMERIC_STD.all;
entity testbench is
end entity test_bench; -- testbench vs. test_bench

architecture BENCH of testbench is
  signal MM : STD_LOGIC;
  signal A, B, C : STD_LOGIC_VECTOR(2 downto 0);
begin
  Stimuli: process
  begin
    for i in 0 to 2**7-1 loop 
      wait for 10 NS;
    end loop;
    wait;
end process Stim; -- Stimuli vs. Stim

end architecture bench_test; -- BENCH vs. bench_test

