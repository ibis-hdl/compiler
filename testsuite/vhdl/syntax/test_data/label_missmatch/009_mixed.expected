in file label_missmatch/009_mixed, line 4:
syntax error: Label mismatch in Entity Declaration
entity testbench is
       ~~~~~~~~~ <<-- here
...
end entity test_bench; -- testbench vs. test_bench
           ~~~~~~~~~~ <<-- and here
in file label_missmatch/009_mixed, line 7:
syntax error: Label mismatch in Architecture Body
architecture BENCH of testbench is
             ~~~~~ <<-- here
...
end architecture bench_test; -- BENCH vs. bench_test
                 ~~~~~~~~~~ <<-- and here
in file label_missmatch/009_mixed, line 11:
syntax error: Label mismatch in Process Statement
  Stimuli: process
  ~~~~~~~ <<-- here
...
end process Stim; -- Stimuli vs. Stim
            ~~~~ <<-- and here
